state-chain/pallets/cf-ingress-egress/src/migrations/deposit_channels_with_boost_fee.rs

```diff
@@ -0,0 +1,217 @@
+use crate::{Instance1, Instance2, Instance3, *};
+use cf_chains::{
+	btc::ScriptPubkey, dot::PolkadotAccountId, Bitcoin, DepositChannel, Ethereum, Polkadot,
+};
+use frame_support::traits::OnRuntimeUpgrade;
+pub struct Migration<T: Config<I>, I: 'static>(PhantomData<(T, I)>);
+
+mod old {
+	use super::*;
+
+	#[derive(
+		CloneNoBound, RuntimeDebug, PartialEq, Eq, Encode, Decode, TypeInfo, MaxEncodedLen,
+	)]
+	#[scale_info(skip_type_params(T, I))]
+	pub struct DepositChannelDetails<T: Config<I>, I: 'static> {
+		pub deposit_channel: DepositChannel<T::TargetChain>,
+		pub opened_at: TargetChainBlockNumber<T, I>,
+		pub expires_at: TargetChainBlockNumber<T, I>,
+		pub action: ChannelAction<T::AccountId>,
+	}
+
+	#[frame_support::storage_alias]
+	pub type DepositChannelLookup<T: Config<I>, I: 'static> = StorageMap<
+		Pallet<T, I>,
+		Twox64Concat,
+		TargetChainAccount<T, I>,
+		old::DepositChannelDetails<T, I>,
+		OptionQuery,
+	>;
+}
+
+impl<T: Config<Instance1, TargetChain = Ethereum>> OnRuntimeUpgrade for Migration<T, Instance1> {
```

Comment:
There's no need to define these separately, you can just do it like this:
```
impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
```

The only reason we had to do that in another place is because we wanted to do something different, for each chain, but here, it's exactly the same